### DISCUSSION TITLE
Unable to view contributions for organizations that use SSO

### DISCUSSION THREAD

**Discussion by SergioB-dev** [community/community#136661] [Other Feature Feedback, Questions, & Ideas] [answered]
### Contributions

The [GitHub documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile) on this says:

> If you are part of an organization that uses SAML single sign-on (SSO), you won’t be able to see contribution activity from the organization on your profile if you do not have an active SSO session.

I imagine a good portion of companies use SSO. So many users' profile still cannot display their private contributions. Is there anyway around this?

**Comment by ebndev** [comment, marked as answer]
Hi @SergioB-dev, thanks for being a part of the GitHub Community!

Did you mean being able to display private contributions in the contribution graph or in the following list that displays i.e: "22 contributions in private repositories"?

If you want to have the contributions count towards your graph, you can adjust your `Contribution settings`. You'll see at the top of the graph also shows the prompt to start an active SSO session.

  <img width="659" alt="Screenshot 2024-08-23 at 11 31 18 AM" src="https://github.com/user-attachments/assets/2003b5ff-f16b-418f-9995-945403c8ad5c">

The contribution number changes for me after signing in, but also stays at that number when other users visit my profile.

  <img width="777" alt="Screenshot 2024-08-23 at 11 36 33 AM" src="https://github.com/user-attachments/assets/9c2ca9d7-2438-42f7-9b8e-3620b49c6c16">

**Reply by SergioB-dev** [reply]
Oh wow, I didn't know an active SSO session would make the contributions visible to visitors. Great, thanks! 

This makes me wonder. Has your team considered allowing this to work without an active SSO session? Or to phrase the question another way, should an active SSO session really block visitors from seeing contributions?

**Reply by ebndev** [reply]
Because I turned on `Private contributions`, visitors can see the total number of contributions without an active SSO!

The top photo was only for my view when I was signed in but not with SSO... is that what you meant? 😅

**Reply by SergioB-dev** [reply]
Hi sorry for my delayed answer, I had some trouble finding this ticket 😅 

@ebndev you definitely answered my question.

My followup question was more of a high level question for GH's roadmap for this feature. Do you know if there are any plans to enable seeing contributions that may be behind SSO **without actually being signed in via SSO**. But the more I think about the more I can see the complications with that.

So thanks for your help!
